Cargo.toml

```diff
@@ -19,15 +19,16 @@ default-target = "x86_64-apple-darwin"
 rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
-block = "0.1.6"
-core-foundation = { version = "0.9", features = ["with-chrono"] }
-core-graphics = "0.22"
+objc = { version = "=0.3.0-beta.1", package = "objc2" }
```

Comment:
I'm fairly close to releasing a stable version of `objc2`, but using it in a bigger project like `cacao` is part of that process, so I would like to hold off on that until we've integrated it more fully, and can more easily see which (if any) breaking changes I'll have to make to `objc2` to make things work more smoothly

Reply:
Of course, this will probably hinder a stable `cacao` release, so I'll keep that in mind and try to move forwards on a stable `objc2` fairly quickly